dist/XSLoader/t/XSLoader.t

```diff
@@ -1,6 +1,8 @@
 #!perl -T
 
 use strict;
+use warnings;
```

Comment:
This introduces warning during the test process:
```
Subroutine XSLoader::bootstrap_inherit redefined at t/XSLoader.t line ***20.
Subroutine XSLoader::bootstrap_inherit redefined at t/XSLoader.t line ***42.
```

Reply:
Good catch, fixed that.